src/app/routes/utils/fetchDataFromBFF/index.ts

```diff
@@ -1,17 +1,15 @@
-import getAgent from '../../../../server/utilities/getAgent';
 import constructPageFetchUrl from '../constructPageFetchUrl';
 import getEnvironment from '../getEnvironment';
 import { Services, Variants, PageTypes } from '../../../models/types/global';
 import fetchPageData from '../fetchPageData';
 import getErrorStatusCode from '../fetchPageData/utils/getErrorStatusCode';
 import { BFF_FETCH_ERROR } from '../../../lib/logger.const';
-import { FetchError } from '../../../models/types/fetch';
+import { FetchError, GetAgent } from '../../../models/types/fetch';
 import nodeLogger from '../../../lib/logger.node';
+import certsRequired from '../certsRequired';
 
 const logger = nodeLogger(__filename);
-const BFF_IS_LOCAL =
-  process.env.JEST_WORKER_ID === undefined &&
-  process?.env?.BFF_PATH?.includes('localhost:3210');
+const BFF_IS_LOCAL = process?.env?.BFF_PATH?.includes('localhost:3210');
```

Comment:
Can this now be removed? `BFF_IS_LOCAL` is used in an if statement below to determine whether to add optional headers or not, but wondering if we could use `certsRequired` instead?

Reply:
I've updated the logic now to try and make it a bit simpler, inline with your suggest. https://github.com/bbc/simorgh/pull/11266/commits/a32bff885d957866ad27dfbfa451649d14981c49

It has changed the value of the headers when running locally, but I tested this with the BFF running on the Test env and with the BFF running locally as well and it seems fine?

Reply:
Couple more commits to get it working, but think its a bit better? Not sure of the need of the `accept` header locally, so its removed.